lib/ui/painting/immutable_buffer.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_LIB_UI_PAINTNIG_IMMUTABLE_BUFER_H_
+#define FLUTTER_LIB_UI_PAINTNIG_IMMUTABLE_BUFER_H_
+
+#include <cstdint>
+
+#include "flutter/fml/macros.h"
+#include "flutter/lib/ui/dart_wrapper.h"
+#include "third_party/skia/include/core/SkData.h"
+#include "third_party/tonic/dart_library_natives.h"
+#include "third_party/tonic/logging/dart_invoke.h"
+#include "third_party/tonic/typed_data/typed_list.h"
+
+namespace flutter {
+class ImmutableBuffer : public RefCountedDartWrappable<ImmutableBuffer> {
+ public:
+  ~ImmutableBuffer() override = default;
```

Comment:
OOL this please. This applies to all virtual methods elsewhere as well.

Reply:
done